<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-J9THCFRO" ignored="3" name="Test Cases(failed)" tests="1" failures="1" timestamp="2021-12-22T12:47:55 IST" time="0.26" errors="0">
  <testcase name="secondTestCase" time="0.221" classname="TestNG.sampleTestNG">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method TestNG.sampleTestNG.secondTestCase() didn&amp;apos;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestNG.sampleTestNG.secondTestCase() didn't finish within the time-out 200
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- secondTestCase -->
  <testcase name="fourthTestCase" time="0.0" classname="TestNG.sampleTestNG">
    <ignored/>
  </testcase> <!-- fourthTestCase -->
  <testcase name="firstTestCase" time="0.0" classname="TestNG.sampleTestNG">
    <ignored/>
  </testcase> <!-- firstTestCase -->
  <testcase name="thirdTestCase" time="0.0" classname="TestNG.sampleTestNG">
    <ignored/>
  </testcase> <!-- thirdTestCase -->
</testsuite> <!-- Test Cases(failed) -->
